packages/outline-view/src/browser/outline-view-tree-model.ts

```diff
@@ -15,7 +15,7 @@
  ********************************************************************************/
```

Comment:
Unfortunately renaming this file may cause breaking changes for downstream extensions and applications.
I understand why it was done, but can you add a note in the `changelog`?

Reply:
Thanks for pointing our the file rename is potentially a breaking change. I've updated the changelog accordingly.